src/firefly/js/visualize/Band.js

```diff
@@ -7,6 +7,10 @@ import Enum from 'enum';
  * @typedef Band
  * A Band in an image plot
  * must be enum 'RED', 'GREEN', 'BLUE', 'NO_BAND'
+ * @prop RED
+ * @prop GREEN
+ * @prop BLUE
+ * @prop NO_BAND
  * @public
  * @global
  * @prop value
```

Comment:
This is in the doc and it shows up as untyped typedef with properties.  You mentioned this is the way to document enum in jsDoc?

Reply:
Some of them I forgot to include `@type {Enum}`. That is what gives more clean in the documentation but just doing typedef satisfies IntelliJ. So I forgot to put `@type` in some of them.

I will fix it.